public/projects/index.html

```diff
@@ -0,0 +1,231 @@
+    <div class="wrapper__content">
+
+      <div class="section section--statechannels">
+        <div class="container">
+
+          <div class="current">
+
+            <h2 class="hero__heading">Current Projects</h2>
+
+            <p class="hero__body">
+              Main contributors to State Channels, an open source project in pursuit of a single standard that enable a
+              fully interoperable ecosystem of State Channel applications. Learn more at <a target="_blank"
+                href="https://www.statechannels.org/">statechannels.org</a>!
+            </p>
+            <p class="hero__body">
+              With support from Filecoin Grants, we are developing Nitro protocol in golang for use in the Filecoin
+              ecosystem. This work lays the foundation of utilizing virtual channels in the Retrieval Market. Check out
+              our <a href="https://github.com/filecoin-project/devgrants/issues/348">grant proposal</a>.
+            </p>
+            <p class="hero__body">
+              [TBD] SAFE placeholder and Nitro V2
+            </p>
+<br>
+          </div>
+
+        </div>
+        <div class="container">
+          <div class="past">
+            <h2 class="hero__heading">State Channel Applications</h3>
+
+              <p class="hero__body">
+                Here at Magmo, we are working towards our goal of building state channel networks, where users can
+                interact off-chain with security —
+                but with minimal cost and latency. To stress-test our protocols and other software, we build
+                applications that utilize them: for example, games!
+              </p>
+              <ul>
+                  <li class="hero__body__list">
+                    <a target="_blank" href="https://rps.magmo.com/">Rock Paper Scissors</a>
+                  </li>
+                  <li class="hero__body__list">
+                    <a target="_blank" href="https://ttt.magmo.com/">Tic Tac Toe</a>
+                  </li>
+              </ul>
+              <p class="hero__body">
+                For a more practical application to showcase the value of state channels, we built <a target="_blank"
+                  href="https://web3torrent.statechannels.org/">Web3Torrent</a> proof of concept application.
+                This browser based torrenting client demonstates the use of peer to peer state channel relationships,
+                high frequency messaging and the ability to incentivize filesharing using micropayments
+              </p>
+              <p class="hero__body">
+                The process of building these application, as well as feedback we get from gamers and devs, is a great
+                opportunity to learn how we could be doing things differently or better.
+              </p>
+              <br>
+              <p class="hero__subheading__right">
+                Check out our <a target="_blank" href="https://github.com/magmo">github</a> repository!
```

Comment:
Might not want to point to GitHub.com/magmo, since we basically don't use it anymore.